src/lib/actions.ts

```
@@ -0,0 +1,40 @@
+
+import { prisma } from "@/lib/prisma"
+import { UserSchema } from "@/lib/utils"
+
+const FirstLoginSchema = UserSchema.pick({
+  username: true,
+  name: true,
+  email: true,
+})
+
+export async function firstLogin(
+  prevState: z.ZodError<z.infer<typeof FirstLoginSchema>> | void,
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)